.github/workflows/docs.yml

```diff
@@ -22,7 +22,6 @@ jobs:
 
       - name: Install apt packages
         run: |
-          sudo apt-get update && sudo apt-get upgrade
```

Comment:
@alkino added something similar last week for some issue?

Reply:
Yes, mainstream `libcaca?` was failing. Should've been stated as temporary